include/yboard.h

```diff
@@ -102,16 +103,66 @@ class YBoardV3 {
      * representing the name of the sound file to play. The return type is a boolean
      * value (true or false). True corresponds to the sound being played
      * successfully, and false corresponds to an error playing the sound. The sound
-     * file must be stored on the microSD card. After this function is called, the
-     * loop_speaker function must be called often to playback the sound on the speaker.
+     * file must be stored on the microSD card.
      */
     bool play_song_from_sd(const char *filename);
```

Comment:
I know this was the name of the function I came up with, but I am wondering if there is a better name. Just `play_song`? Is the SD card implied? What do you think?

Reply:
I agree a shorter name would be good, and SD card is implied.  Maybe `play_sound_file`?  `play_song` is fine too, but often they aren't really playing "songs".

Reply:
I like `play_sound_file`